### ISSUE TITLE
IOOS 1.2 `NamingAuthorityValidator` error

### ISSUE THREAD

**Issue by daltonkell** [ioos/compliance-checker#746]
Upon calling the `ioos:1.2` check on this [SECOORA dataset](http://erddap.secoora.org/erddap/info/edu_usf_marine_comps_1407d550/index.html), an exception is caught:

```
WARNING: The following exceptions occured during the ioos:1.2 checker (possibly indicate compliance checker issues):
ioos:1.2.check_high: 'NamingAuthorityValidator' object is not callable
```

Side note: "occurred" is spelled incorrectly in the above message.

**Comment by benjwadams** [comment]
@daltonkell, are you running off of latest master?  This was an oversight on my part and should be fixed in 0668c337e96578eaef2cc734ebd2bfe231284cc4 and 71b94963bbd1574b7517ea91ca025799119f55fd.

**Comment by daltonkell** [comment]
The wonders of `git fetch`
